cli/shell.go

```diff
@@ -0,0 +1,51 @@
+package run
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func bashShell(cluster string, command string) error {
+	kubeConfigPath, err := getKubeConfig(cluster)
+	if err != nil {
+		return err
+	}
+
+	subShell := os.ExpandEnv("$__K3D_CLUSTER__")
+	if len(subShell) > 0 {
+		return fmt.Errorf("Error: Already in subshell of cluster %s", subShell)
+	}
+
+	bashPath, err := exec.LookPath("bash")
+	if err != nil {
+		return err
+	}
+
+	cmd := exec.Command(bashPath, "--noprofile", "--norc")
+
+	if len(command) > 0 {
+		cmd.Args = append(cmd.Args, "-c", command)
+
+	}
+
+	// Set up stdio
+	cmd.Stdout = os.Stdout
+	cmd.Stdin = os.Stdin
+	cmd.Stderr = os.Stderr
+
+	// Set up Promot
+	setPS1 := fmt.Sprintf("PS1=[%s}%s", cluster, os.Getenv("PS1"))
```

Comment:
This is the only part that doesn't work for me, but that's probably due to conflicts between `PS1` setup in my `zsh` and this one

Reply:
It seems you are zsh user. I'd love to have you contribute a solution that works with it -- I think zsh is a popular shell.